pilot/pkg/config/coredatamodel/syntheticserviceentrycontroller.go

```diff
@@ -270,31 +270,25 @@ func (c *SyntheticServiceEntryController) configStoreUpdate(resources []*sink.Ob
 			svcChangeByNamespace[conf.Namespace] = struct{}{}
 			continue
 		}
-
-		// this is done before updating internal cache
-		oldEpVersion := c.endpointVersion(conf.Namespace, conf.Name)
-		newEpVersion := version(conf.Annotations, endpointKey)
-		if oldEpVersion != newEpVersion {
-			if err := c.edsUpdate(conf); err != nil {
```

Comment:
why remove this?

Reply:
because it is inaffective now -  any update needs a full push

Reply:
Didn't we only do eds push when svc not changed?

Reply:
Yes. That is what we have to do. But it is not working for SSE. See the TODO below and  #18625